### DISCUSSION TITLE
17-01-2024 Steering Committee Meeting Minutes and Resolutoin

### DISCUSSION THREAD

**Discussion by hackfisher** [ringdao-community-guild/discussion#67] [Steering Committee]
1. Review and approve Cindy resignation as Community Manager due to health reasons, and discuss the handover arrangements, including the issue of successor candidates (it is proposed to find the candidates first from ambassador program, senior ambassador or ambassador).
Note: Special thanks to Cindy for her contributions as the Community Manager to the DAO. Wishing her continued health and well-being.

2. Nominations and elections for the next Steering Committee will be taken in advance.
- Nominations Period: First day in Feb - Last day in March
- Nominations Method: You must be a member, apply by creating a new discussion in the "Steering Committee" category express your intention to run for Committee. Introduce yourself to gain support from others.
- Voting Period: First day in April - Last day in May
- Voting Method: All candidates will be collected from the forum, and organised in a candidates list in a proposal for all the members to voting, will adopt Ranked choice voting(ref: [example](https://snapshot.org/#/pistachiodao.eth/proposal/0x5003da0f03e718b461e53fe10a998b60172e2e108472153282fcef781c300f23)) for the voting type. There are 5 positions or options to be elected (like in a committee election or multiple policy decisions), voters can still rank their choices. The counting process then considers these rankings to allocate the seats or decide on the options based on the collective preferences.

- Another proposal will be initiated to request a small amount of funding for compensating and incentivizing the Steering Committee for the time invested in attending meetings.
- If a committee member exits during the term, candidates from the election will automatically fill the vacancy based on voting results. If there are no remaining candidates, former committee members from the previous year who are not serving this year will automatically fill the position.

3. Begin using the new domain dcdao3.eth and the new email address hello.dcdao3.eth@skiff.com 
Note: The NFT of dcdao3.eth is owned by the DAO's treasury address (safe multisig) on Ethereum. hello.dcdao3.eth@skiff.com is allocated to one member who is managing the public media resources such as twitter. (Current DAO's twitter email settings should be changed to hello.dcdao3.eth@skiff.com)

**Comment by Laki389** [comment]
> 1. Review and approve Cindy resignation as Community Manager due to health reasons, and discuss the handover arrangements, including the issue of successor candidates (**it is proposed to find the candidates first from ambassador program, senior ambassador or ambassador**).
>        Note: Special thanks to Cindy for her contributions as the Community Manager to the DAO. Wishing her continued health and well-being.
> 

- I wish Cindy good health and all the best in life and in her future career.

- Since it is proposed that the future Community Manager should first be sought from among the current (Sen.) Ambassadors, IMHO:

I think the best for that position out of the 4 of us is **Jesse**, he will be great for that position, I encourage and ask him to apply so that we as DCDAO and as a community can move forward,

DCDAO, FTW!

**Reply by hackfisher** [reply]
Agree， @1022ranji Please consult with relevant personnel and facilitate this matter, thanks

**Reply by SasoLithops** [reply]
I absolutely agree with Laki and support Jesse as community manager. I have only but huge respect for him and he has the best technical knowledge,many friends in the ecosystem, a lot of experiences and has been the one we always turn too for answers.
